### PR TITLE
fix import delete depricated fix linters

### PR DESCRIPTION
**Pull request by pfi79**

### CODE REVIEW

File: internal/bft/batcher.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func (b *BatchBuilder) NextBatch() [][]byte {
 		return currBatch
 	}
 
-	timeout := time.After(b.batchTimeout) //TODO use task-scheduler based on logical time
+	timeout := time.After(b.batchTimeout) // TODO use task-scheduler based on logical time
```
Comment (C0rWin) [inline internal/bft/batcher.go]: Is this change after gofmt?

File: internal/bft/batcher_test.go [hunk 2]
```diff
@@ -18,9 +18,7 @@ import (
 	"go.uber.org/zap"
 )
 
-var (
-	noopTimeoutHandler = &mocks.RequestTimeoutHandler{}
-)
+var noopTimeoutHandler = &mocks.RequestTimeoutHandler{}
```
Comment (C0rWin) [inline internal/bft/batcher_test.go]: Is it gofmt change?

File: pkg/api/dependencies.go [hunk 3]
```diff
@@ -36,7 +36,7 @@ type Assembler interface {
 	AssembleProposal(metadata []byte, requests [][]byte) bft.Proposal
 }
 
-// WriteAheadLog is a write ahead log.
+// WriteAheadLog is wroted ahead log.
```
Comment (C0rWin) [inline pkg/api/dependencies.go]:
```suggestion
// WriteAheadLog is write ahead log.
```

the previous comment was correct

File: pkg/consensus/consensus.go [hunk 4]
```diff
@@ -443,15 +441,12 @@ func (c *Consensus) setViewAndSeq(view, seq, dec uint64) (newView, newSeq, newDe
...
-		// Check if the view change has a newer view
-		if viewChange.NextView >= view {
-			c.Logger.Debugf("Restoring from view change with view %d", viewChange.NextView)
-			newView = viewChange.NextView
-			restoreChan := make(chan struct{}, 1)
-			restoreChan <- struct{}{}
-			c.viewChanger.Restore = restoreChan
-		}
+	} else if viewChange.NextView >= view {
+		c.Logger.Debugf("Restoring from view change with view %d", viewChange.NextView)
```
Comment (C0rWin) [inline pkg/consensus/consensus.go]: Please keep inline comment

File: pkg/consensus/consensus.go [hunk 5]
```diff
@@ -460,14 +455,11 @@ func (c *Consensus) setViewAndSeq(view, seq, dec uint64) (newView, newSeq, newDe
...
-	} else {
-		// Check if metadata should be taken from the restored new view
-		if viewSeq.Seq >= seq {
-			c.Logger.Debugf("Restoring from new view with view %d and seq %d", viewSeq.View, viewSeq.Seq)
-			newView = viewSeq.View
-			newSeq = viewSeq.Seq
-			newDec = 0
-		}
+	} else if viewSeq.Seq >= seq {
+		c.Logger.Debugf("Restoring from new view with view %d and seq %d", viewSeq.View, viewSeq.Seq)
```
Comment (C0rWin) [inline pkg/consensus/consensus.go]: please keep inline code comment

**Review by C0rWin — CHANGES_REQUESTED** [review]
There are a few changes here:

1. Linters
2. Code style
3. Switching to new go std functions

Could you please split the PR into three different?

File: internal/bft/batcher.go (hunk 1 above)
Comment (pfi79) [reply internal/bft/batcher.go]: yes

File: internal/bft/batcher_test.go (hunk 2 above)
Comment (pfi79) [reply internal/bft/batcher_test.go]: no

File: pkg/consensus/consensus.go (hunk 5 above)
Comment (pfi79) [reply pkg/consensus/consensus.go]: corrected

File: pkg/consensus/consensus.go (hunk 4 above)
Comment (pfi79) [reply pkg/consensus/consensus.go]: corrected

File: pkg/api/dependencies.go (hunk 3 above)
Comment (pfi79) [reply pkg/api/dependencies.go]: corrected

**Pull request closed without merging**